components/Header/index.js

```diff
@@ -9,4 +9,37 @@
 //  </div >
 // And add it to the DOM in the .headerContainer component
 
-function Header() {}
+function Header() {
+
+    const divCreator = document.createElement('div');
```

Comment:
creator might not be the best way to name the function here.  As it only made it once.